rocketpool-cli/node/stake-rpl.go

```diff
@@ -187,38 +190,58 @@ func nodeStakeRpl(c *cli.Context) error {
 		amountWei = eth.EthToWei(stakeAmount)
 
 	} else {
-
-		// Get min/max per minipool RPL stake amounts
-		rplPrice, err := rp.RplPrice()
-		if err != nil {
-			return err
-		}
-		minAmount8 := rplPrice.MinPer8EthMinipoolRplStake
+		// Get the RPL stake amounts for 5,10,15% borrowed ETH per LEB8
+		fivePercentBorrowedPerMinipool := new(big.Int)
+		fivePercentBorrowedPerMinipool.SetString("50000000000000000", 10)
+		fivePercentBorrowedRplStake := borrowedRplStake(fivePercentBorrowedPerMinipool, rplPrice.RplPrice)
+		tenPercentBorrowedRplStake := new(big.Int).Mul(fivePercentBorrowedRplStake, big.NewInt(2))
+		fifteenPercentBorrowedRplStake := new(big.Int).Mul(fivePercentBorrowedRplStake, big.NewInt(3))
 
 		// Prompt for amount option
 		amountOptions := []string{
-			fmt.Sprintf("The minimum minipool stake amount for an 8-ETH minipool (%.6f RPL)?", math.RoundUp(eth.WeiToEth(minAmount8), 6)),
+			fmt.Sprintf("5%% of borrowed ETH (%.6f RPL) for one minipool?", math.RoundUp(eth.WeiToEth(fivePercentBorrowedRplStake), 6)),
+			fmt.Sprintf("10%% of borrowed ETH (%.6f RPL) for one minipool?", math.RoundUp(eth.WeiToEth(tenPercentBorrowedRplStake), 6)),
+			fmt.Sprintf("15%% of borrowed ETH (%.6f RPL) for one minipool?", math.RoundUp(eth.WeiToEth(fifteenPercentBorrowedRplStake), 6)),
 			fmt.Sprintf("Your entire RPL balance (%.6f RPL)?", math.RoundDown(eth.WeiToEth(&rplBalance), 6)),
 			"A custom amount",
+			"A custom percentage",
```

Comment:
Let's fold this into "A custom amount"...

Again, just tell the user to either enter an amount of rpl or a precentage of borrowed eth, and check if there's a `%` suffix